grebase.sh

```diff
@@ -21,6 +21,7 @@ if [ -z "$cur_dev" ]
 	    echo "Failed to get develop!" 1>&2
 	exit 1
 fi
+git stash -- site.properties
```

Comment:
I didn't test this script but happy to learn how to use it, looks quite useful to me.